app/src/main/java/com/simplecity/amp_library/ui/presenters/PlayerPresenter.java

```diff
@@ -124,11 +129,15 @@ public void bindView(@NonNull PlayerView view) {
     }
 
 
-    private void refreshCurrentTimeText(long playbackTime) {
+    private void refreshTimeText(long playbackTime) {
         if (playbackTime != currentPlaybackTime) {
             PlayerView view = getView();
             if (view != null) {
                 view.currentTimeChanged(playbackTime);
+                if(SettingsManager.getInstance().preferRemainingTime())
+                    view.totalTimeChanged(totalTime/1000 - playbackTime);
+                else
+                    view.totalTimeChanged(totalTime/1000);
```

Comment:
Do we need to call `totalTimeChanged()` in the `else` case here? If we're not showing 'remaining time', then the total time text only changes in `updateTrackInfo()` (as opposed to every 150ms when the current time updates). 

If you do remove the else case, we're going to need a way to make sure the text updates if the user changes their 'show remaining time' preference. This is probably best achieved via a preference listener.